lib/modules/smart_scheduling_links/smart_scheduling_links_basic_sequence.rb

```diff
@@ -776,16 +776,6 @@ def get_file(file, use_token: true)
                 end
             @invalid_service_type_count += 1
           end
-
-          @invalid_vaccine_product_count += 1 if resource.extension.nil? || resource.extension.none? do |extension|
-            extension.url == 'http://fhir-registry.smarthealthit.org/StructureDefinition/vaccine-product' &&
-            !extension.valueCoding.nil?
-          end
-
```

Comment:
If you remove these blocks, you need to remove the tests as well...

I'm going to undelete these for now (and fix the _product bug) and merge, because if we remove these we really need to go through and remove them thoroughly, *and* remove the other redundant tests.  But I would like to get in the other improvement for now.